locales/en/package.i18n.json

```diff
@@ -12,5 +12,7 @@
   "deviceSimulatorExpressExtension.commands.microbit.openSimulator": "[micro:bit] Open Simulator",
   "deviceSimulatorExpressExtension.commands.microbit.newFile": "[micro:bit] New File",
   "deviceSimulatorExpressExtension.configuration.title": "Device Simulator Express configuration",
-  "deviceSimulatorExpressExtension.configuration.properties.debuggerPort": "The port the Server will listen on for communication with the debugger."
-}
+  "deviceSimulatorExpressExtension.configuration.properties.configEnvOnChange": "When you change the Python Interpreter, the dependencies for the Device Simulator Express will automatically be configured for the extension dependencies (and the interpreter will auto-change if necessary).",

```

Comment:
It might be confusing for the users if you add that the interpreter will auto change if necessary, but then later you ask them to ctrl shift p to change the interpreter manually

Reply:
Everything only works if the original interpreter version is correct. Do you think I should add that to this description?

Reply:
description updated for clarity!